src/camera-controls.js

```diff
@@ -882,6 +902,45 @@ export default class CameraControls extends EventDispatcher {
 
 	}
 
+	_encloseToBoundary( position, offset, friction ) {
+
+		const offsetLength = offset.length();
+
+		if ( offsetLength === 0.0 ) { // sanity check
+
+			return position;
+
+		}
+
+		const t1 = _v3B.copy( offset ).add( position ); // target
+		const tc = this.boundary.clampPoint( t1, _v3C ); // clamped target
+		const dtc = tc.sub( t1 ); // t1 -> tc
+		const dtcl = dtc.length(); // length of dtc
```

Comment:
Full-form name is preferable rather than abbreviated.

Reply:
I used mathematic oriented notation here, and also scope is pretty short so I thought it's fine,
but if I need to rename these, I'd choose `newTarget` , `clampedTarget` , `deltaClampedTarget` , `deltaClampedTargetLength`

Reply:
That makes sense. But if you could rename to those, should be better for other users who may read the code🙏

Reply:
Okay 'n' done

Reply:
Thanks a lot!